src/System.Net.Http/src/System/Net/Http/SocketsHttpHandler/BaseAsyncStream.cs

```diff
@@ -3,12 +3,13 @@
 // See the LICENSE file in the project root for more information.
 
 using System.IO;
+using System.Runtime.InteropServices;
 using System.Threading;
 using System.Threading.Tasks;
 
 namespace System.Net.Http
 {
-    internal abstract class BaseAsyncStream : Stream
+    internal abstract class HttpBaseStream : Stream
```

Comment:
If you're renaming the class, the filename should be renamed to match.

Reply:
Good point, thanks.